modules/local/fusioninspector/main.nf

```diff
@@ -4,8 +4,8 @@ process FUSIONINSPECTOR {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/be/bed86145102fdf7e381e1a506a4723676f98b4bbe1db5085d02213cef18525c9/data' :
-        'community.wave.seqera.io/library/dfam_hmmer_minimap2_star-fusion:aa3a8e3951498552'}"
+        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/7d/7dbc3bb67499701633a925b0ed03f58ffb03d0a4c37a86a53c38ebe9a8a6cc37/data' :
```

Comment:
This container does not seem to contain a fusioninspector executable

Reply:
Mmm, that's odd. I will run the nf-test again in singularity and check.

Reply:
The test did not fail for me but I made a few changes that could have been causing issues if you were trying to replicate it:
- Removed a hardcoded `starfusion_build` invocation in the nf-test
- Added a fixed version of `conda-forge::perl-carp-assert=0.21`

Let me know if you are still encountering issues.